src/libs/actions/PaymentMethods.ts

```diff
@@ -185,19 +204,20 @@ function addPaymentCard(params) {
 function clearDebitCardFormErrorAndSubmit() {
     Onyx.set(ONYXKEYS.FORMS.ADD_DEBIT_CARD_FORM, {
         isLoading: false,
-        errors: null,
+        errors: undefined,
+        setupComplete: true,
     });
 }
 
+type TransferWalletBalanceParameters = Partial<Record<ValueOf<typeof CONST.PAYMENT_METHOD_ID_KEYS>, number | undefined>>;
```

Comment:
You can move params type to function body (examples in Fabio's [PR](https://github.com/Expensify/App/pull/28269))